docs/library/graphing/plotly.md

```diff
@@ -100,14 +46,55 @@ z_data = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/a
 fig = go.Figure(data=[go.Surface(z=z_data.values)])
 fig.update_traces(contours_z=dict(show=True, usecolormap=True,
                                   highlightcolor="limegreen", project_z=True))
-fig.update_layout(scene_camera_eye=dict(x=1.87, y=0.88, z=-0.64),
-                  width=500, height=500,
-                  margin=dict(l=65, r=50, b=65, t=90)
+fig.update_layout(
+    scene_camera_eye=dict(x=1.87, y=0.88, z=-0.64),
+    width=500, height=500,
+    margin=dict(l=65, r=50, b=65, t=90)
 )
+
+def mountain_surface():
+    return rx.center(
+        rx.plotly(data=fig),
+    )
 ```
 
-Now pass the plotly figure again to the plotly component.
+## Plot as State Var
```

Comment:
For this graph we should add more graph types and enable them to be shifted through using a select. A bar chart and a scatter plot as well.

Reply:
WIP For next release.